controller_interface/CMakeLists.txt

```diff
@@ -6,7 +6,6 @@ find_package(catkin REQUIRED COMPONENTS roscpp hardware_interface pluginlib)
 
 # Declare catkin package
 catkin_package(
-  DEPENDS boost
```

Comment:
`controller_manager` uses `Boost` here: https://github.com/ros-controls/ros_control/blob/kinetic-devel/controller_manager/src/controller_manager.cpp#L117

but `controller_interface` does NOT use it at all.

So, you should remove it here (`controller_interface`) and fix it for `controller_manager`.